packages/types/src/modules-sdk/index.ts

```diff
@@ -168,7 +168,7 @@ export type ModuleJoinerConfig = Omit<
 }
 
 export declare type ModuleJoinerRelationship = JoinerRelationship & {
-  isInternalService?: boolean // If true, the relationship is an internal service from the medusa core
+  isInternalService?: boolean // If true, the relationship is an internal service from the medusa core TODO: Remove once we get to the point hat there is more internal services
```

Comment:
You mean no more right? 😄 
```suggestion
  isInternalService?: boolean // If true, the relationship is an internal service from the medusa core TODO: Remove when there are no more "internal" services
```